app/code/Magento/Review/Block/Product/ReviewRenderer.php

```diff
@@ -33,17 +35,25 @@ class ReviewRenderer extends \Magento\Framework\View\Element\Template implements
      */
     protected $_reviewFactory;
 
+    /**
+     * @var ReviewSummaryFactory
+     */
+    private $reviewSummaryFactory;
+
     /**
      * @param \Magento\Framework\View\Element\Template\Context $context
      * @param \Magento\Review\Model\ReviewFactory $reviewFactory
+     * @param ReviewSummaryFactory $reviewSummaryFactory
      * @param array $data
      */
     public function __construct(
         \Magento\Framework\View\Element\Template\Context $context,
         \Magento\Review\Model\ReviewFactory $reviewFactory,
+        ReviewSummaryFactory $reviewSummaryFactory,
```

Comment:
Please add new constructor parameters in a backward compatible way: as a last optional parameter.
See more details in [Backward compatible development guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#php) "Adding Constructor parameters" section.

Reply:
Fixed